script.js

```diff
@@ -0,0 +1,59 @@
+const title = document.querySelector('#inputTitle');
+const author = document.querySelector('#inputAuthor'); // eslint-disable-line no-undef
+const bookList = document.querySelector('#bookList');// eslint-disable-line no-unused-vars
+const add = document.querySelector('#add');
+let books = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
+  books.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(books));
+  return { title, author };
+}
+
+function removeBook(element) {
+  if (element.classList.contains('remove')) {
+    const removeItem = element.parentElement;
+    books = books.filter((book) => book.title !== element);
+    removeItem.remove();
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```

Comment:
- [ ] Currently, your `removeBook` function is removing only the HTML element but not the data of that book, kindly fix this issue 🤓 
    - I would recommend adding an `id` to the book objects so you can track or remove elements easily, currently you are using the name to track them 😄
    - As you can see in this image I have removed the books but they still appear in the local storage, so when I reload the page they will appear at the top of the page again
    ![image](https://user-images.githubusercontent.com/60789317/134965918-ffe2958d-250c-450d-98ac-f75e369567e3.png)